bin/functions.sh

```diff
@@ -19,7 +20,7 @@ DATE_CMD="/bin/date"
 # arguments: 1. s3 url (s3://.../...)
 s3_exists() {
 	if [ $# -ne 1 ]; then return 255; fi
-	${AWSCLI} ${AWSCLIOPT} ${AWSCLI_LIST_OPT} $1 >/dev/null
+	${AWSCLI} ${AWSCLI_ENDPOINT_URL_OPTION} ${AWSCLIOPT} ${AWSCLI_LIST_OPT} $1 >/dev/null
```

Comment:
`AWSCLI_ENDPOINT_URL_OPTION` will be set like `fra1.digitaloceanspaces.com`, so `--endpoint-url` is needed here, right?

Reply:
Yes. It's important to set the whole aws cli parameter "--endpoint-url https://domain.tld".
It is easier to implement this way because if a user wanna use the default S3 endpoint the only thing he must ensure is to not set the AWSCLI_ENDPOINT_URL_OPTION var. That's it. 😃